packages/fonts/src/get-font-faces.ts

```diff
@@ -16,7 +16,8 @@ export type FontFace = {
   fontStretch?: string;
 };
 
-const sanitizeCssUrl = (str: string) => str.replace(/["\\]{1}/g, "\\$&");
+// Use JSON.stringify to escape double quotes and backslashes in strings as it automatically replaces " with \" and \ with \\.
+const sanitizeCssUrl = (str: string) => JSON.stringify(str);
```

Comment:
lol why bother with an extra function then

Reply:
JSON.stringify looks strange, and it looks like beautiful hack for me. Just to give name to this hack